blog/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self):
+        return f"{self.username}"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(User,
+                              on_delete=models.CASCADE,
+                              related_name="posts")
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(User,
+                             on_delete=models.CASCADE,
+                             related_name="commentaries")
+    post = models.ForeignKey(Post,
+                             on_delete=models.CASCADE,
+                             related_name="commentaries")
+    created_time = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
```

Comment:
For the `content` field of the `Commentary` model, you might want to consider adding `blank=False, null=False` to ensure that a comment cannot be created without content.